### DISCUSSION TITLE
I'm unable to use ComponentCustomProperties anymore for helping out my VSCode

### DISCUSSION THREAD

**Discussion by d8vjork** [vuejs/language-tools#1024] [Q&A]
This is my entrypoint app.ts:

```ts
import { trans, asset, reload, route } from "@/util";

declare module '@vue/runtime-core' {
  interface ComponentCustomProperties {
    __: typeof trans;
    asset: typeof asset;
    reload: typeof reload;
    route: typeof route;
  }
}

// createApp here...
```

This doesn't work anymore, I used this in my `globals.ts` which shouldn't matter as it was working before on a previous version of Vue or Volar, don't know now how to make this work again...

**Comment by xiaoxiangmoe** [comment]
Can you provide a minimal reproduction repo?
